lmfit/minimizer.py

```diff
@@ -1415,20 +1448,28 @@ def minimize(fcn, params, method='leastsq', args=None, kws=None,
     fit_kws : dict, optional
         Options to pass to the minimizer being used.
 
-    Notes
```

Comment:
Why were the notes placed at the top of the docstring? When using interactive help people want to see Parameters first, then Returns, then Notes.